addons/beehave/nodes/decorators/decorator.gd

```diff
@@ -5,4 +5,4 @@ class_name Decorator, "../../icons/category_decorator.svg"
 
 func _ready():
 	if self.get_child_count() != 1:
-		push_error("Beehave Error: Decorator %s should have only one child" % self.name)
+		push_error("Beehave Error: Decorator %s (%s) should have only one child" % [self.name, self.get_path])
```

Comment:
you are also missing brackets here

Reply:
I'm sorry but which brackets are you referring to 😅

Reply:
`get_path()` instead of `get_path`

Reply:
Oh those ones, thanks for pointing that out